submissions/onyemeMichael/php-db/customerDb.sql

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Why is this file still here? Kindly use the `customers` table you created in the previous task